website/docs/r/virtual_machine.html.markdown

```diff
@@ -276,6 +276,23 @@ For more information on the different example configurations, please check out t
 * `image_uri` - (Optional) Specifies the image_uri in the form publisherName:offer:skus:version. `image_uri` can also specify the [VHD uri](https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-linux-cli-deploy-templates/#create-a-custom-vm-image) of a custom VM image to clone. When cloning a custom disk image the `os_type` documented below becomes required.
 * `os_type` - (Optional) Specifies the operating system Type, valid values are windows, linux.
 * `disk_size_gb` - (Optional) Specifies the size of the os disk in gigabytes.
+* `encryption_settings` - (Optional) Species the encryption settings for the OS disk as documented below. 
+See [Azure Disk Encryption for Windows and Linux IaaS vms](https://docs.microsoft.com/en-us/azure/security/azure-security-disk-encryption) 
+for more information.
+
+`encryption_settings` supports the following:
+
+* `enabled` - (Required) Specifies if the encryption is enabled. If false, disk_encryption_key and key_encryption_key must not be specified.
```

Comment:
can we add `quote's` around `false`, `disk_encryption_key` and `key_encryption_key`?

Reply:
done